jenkins/container/image.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# expects arch to be in environment
+#
+# signing_key and signer may be absent
+
+set -exuo pipefail
+
+source ci-automation/image.sh
```

Comment:
Given that the `ci-automation/image.sh` probably won't be sourced by something else, how about moving all this into it instead of creating new files?

Reply:
same for packages/vms/sdk(_container)

Reply:
I'll have a look, but I'll let the CI to finish it's test run first.

Reply:
New files are gone now.